packages/commons/src/events/ActionDocument.ts

```diff
@@ -41,9 +42,9 @@ export const ActionBase = z.object({
   createdAt: z.string().datetime(),
   createdBy: z.string(),
   createdByRole: z.string(),
+  createdAtLocation: CreatedAtLocation,
```

Comment:
I'm considering changing this to be `UUID` in my PR. Is there a specific reason why this type is branded?